articles/ux-markup/names.md

```diff
@@ -4,24 +4,32 @@ The `ux:Name` attribute in UX Markup is used to provide a local name to a UX obj
 
 ## Syntax
 
+```xml
 	<type ux:Name="node_name" />
+```
 
 Where `type` is any type accessible in UX Markup, and `node_name` is a valid Uno identifier.
 
 ## Remarks
 
 Any object (XML element) in UX markup can be given a `ux:Name` attribute to assign a local name to it. 
 
+```xml
 	<Panel ux:Name="panel1" />
+```
 
 Once a name is given, the object can be referenced by that name within the same *scope*. Any UX expression can then refer to this object by it's plain name. Local names take precedence over global resources with the same name.
 
+```xml
 	<Rectangle LayoutMaster="panel1" />
+```
 
 Names can be used direclty in UX Expressions, for example like this:
 
+```xml
 	<Rectangle Width="width(panel1) / 2"
-	
+```
```

Comment:
You didn't introduce this, but this seems to be missing the closing tag...

Reply:
What's the problem? There was an extra line

Reply:
oh I see, lol

Reply:
Is this part of our mandate or scope for these commits? 😺

Reply:
No. I just wanted to point it out when I noticed.